packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -0,0 +1,199 @@
+                                        marketType !== 'all'
+                                    ) {
+                                        return (
+                                            <div className='flex gap-300'>
+                                                {!(marketType in verificationDocs)
+                                                    ? verificationDocumentsMapper.notApplicable.icon
+                                                    : verificationDocs[marketType]?.map(doc => (
+                                                          <JurisdictionCardVerificationTag
+                                                              category={verificationDocumentsMapper[doc].category}
+                                                              icon={verificationDocumentsMapper[doc].icon}
+                                                              key={`verification-doc-${doc}`}
+                                                          />
+                                                      ))}
+                                            </div>
+                                        );
+                                    }
+
+                                    if (row?.titleIndicators?.displayText) {
+                                        return (
+                                            <div
+                                                className={qtMerge(
+                                                    'rounded-200 bg-brand-red-darker text-system-light-primary-background px-500 py-[5px]',
+                                                    row.titleIndicators?.displayTextSkinColor === 'red-darker' &&
+                                                        'bg-brand-red-darker',
+                                                    row.titleIndicators?.displayTextSkinColor === 'red-dark' &&
+                                                        'bg-brand-red-dark',
+                                                    row.titleIndicators?.displayTextSkinColor === 'red-light' &&
+                                                        'bg-brand-red-light',
+                                                    row.titleIndicators?.displayTextSkinColor === 'yellow-dark' &&
+                                                        'bg-brand-yellow-dark',
+                                                    row.titleIndicators?.displayTextSkinColor === 'yellow-light' &&
+                                                        'bg-brand-yellow-light',
+                                                    row.titleIndicators?.displayTextSkinColor === 'voilet-dark' &&
+                                                        'bg-brand-voilet-dark',
+                                                    row.titleIndicators?.displayTextSkinColor === 'brown-dark' &&
+                                                        'bg-brand-brown-dark'
+                                                )}
```

Comment:
Used this conditional qtMerge as i dont know any other way to handle this.
Using function is just like dumping this thing into somewhere else in order to have a clean look here, which doesn't make sense. As function should be made if you have a reusable functionality.

Reply:
We can use `cva` for styling different variants and condition. Take a look here [docs](https://cva.style/docs) or checkout [@deriv/quill-design](https://github.com/deriv-com/quill-design) src 😃

Reply:
I think it will be same as qtMerge because we need conditional Colors.

Reply:
<img width="605" alt="image" src="https://github.com/binary-com/deriv-app/assets/120543468/fc2bd2bd-6410-4f87-91fd-e8a0b5660c34">

Reply:
using CVA is cleaner and more readable